## Building this document

To build this README, run `build_readme.R`. Talks data is in csv `talks_table.csv`

## Talks

In alphabetical order.

<strong>Chase Clark</strong> (<i>University of Illinois</i>), Your Missing Step in Reproducible R Programming: Continuous Deployment<details><summary>Abstract</summary></p>The past few years have shown vast improvements in workflows for reproducible and distributable research within the R ecosystem. At satRday Chicago everyone in the audience said they used R Markdown, however only one person raised their hand when asked if they could associate their reports back to the code version that generated it. Since continuous integration is quickly becoming commonplace in the R community, continuous deployment (CD) is a logical and easy step to add to your workflow to enhance reproducibility. I will demo associating R Markdown to the code version that produced it and automating the build and release of both executable and cloud-based Shiny apps. Finally, an announcement of the electricShine package for creating Electron based Shiny apps will highlight the power of using CD with production-level Shiny apps.</p><br>[Slides](https://github.com/rinpharma/rinpharma2019program/tree/master/talks_folder/2019-Clark-Continuous_Deployment.pdf)</details><br>
<strong>David Cooper</strong> (<i>GlaxoSmithKline</i>), Using Machine Learning and Interactive Graphics to Find New Cancer Targets<details><summary>Abstract</summary></p>GlaxoSmithKline is searching for new oncology drug targets.  We have CRISPR knockout data for many cancer cell lines and many genes.  For these same cell lines, we also have genomic data --somatic mutations, copy number variants, and gene expression.  We use machine learning (random forests) to find predictive relationships between genomic features and cell line growth under knockout.  Then we use GLASSES, a shiny app, to share the results with biologists.  GLASSES lets scientists interactively explore key relationships and discover novel cancer vulnerabilities.</p><br>[Slides](https://github.com/rinpharma/rinpharma2019program/tree/master/talks_folder/2019-Cooper-Using_ML_and_Interactive_Graphics_to_Find_New_Cancer_Targets.pptx)</details><br>
<strong>Elena Rantou</strong> (<i>FDA</i>), Using R for Generic Drug Evaluation and SABE R-package for Assessing Bioequivalence of Topical Dermatological Products<details><summary>Abstract</summary></p>Determination of bioequivalence (BE), a crucial part of the evaluation of generic drugs, may depend on clinical endpoint studies, pharmacokinetic (PK) studies of bioavailability, and In-Vitro tests, among others. Additionally, in reviewing Abbreviated New Drug Applications (ANDA), FDA reviewers often analyze safety studies and perform various kinds of simulations. A growing, vibrant group of statisticians in the Office of Biostatistics, CDER/FDA has adopted R for both their routine tasks and to address numerous scientific questions that are received in the form of internal consults. During the past 5 years, we have used R to run power simulations; generate the distribution of certain statistics of interest; assess the similarity of and cluster amino-acid sequences as well as, derive the distribution of the molecular weight of such sequences of a certain length; and determine the validity of data sets categorized for genotoxicity. R-package SABE was developed to accompany a new statistical test, used to assess BE of topical dermatological products when data for evaluation come from the In-Vitro Permeation Test (IVPT) [1]. BE tests consider comparisons between a Test (usually generic) and a Reference (RLD) product under a replicate study design. A function that assesses BE of a Test and a Reference formulation uses a mixed scaled criterion for the PK metrics AUC (Area Under the Curve) and Cmax (maximum concentration).</p><br>[Slides](https://github.com/rinpharma/rinpharma2019program/tree/master/talks_folder/2019-Rantou-Generic_Drug_Evaluation_Bioequivalence_Topical_Dermatological_Products.pdf)</details><br>
<strong>Eric Nantz</strong> (<i>Eli Lilly</i>), Creating and reviving Shiny apps with {golem}<details><summary>Abstract</summary></p>Developing Shiny applications that meet design goals, easily deploy to multiple platforms, and contain easily maintainable components (all while adhering to best practices) is typically a difficult endeavor. Until recently, there has not been a tool addressing the optimal development workflow and construction of Shiny apps. The {golem} package by Think-R offers an opinionated framework for creating a Shiny app as a package, with {usethis}-like functionality to add a diverse set of capabilities. In this presentation, I will share how {golem} enables a robust standard for Shiny development and how it magically brought a dormant application back to life.</p><br>[Slides](https://rpodcast.gitlab.io/golem_rpharma2019/#1)</details><br>
<strong>Jeannine Fisher</strong> (<i>Metrum</i>), Leveraging multiple R tools to make effective pediatric dosing decisions<details><summary>Abstract</summary></p>R Shiny apps allow for dynamic, interactive, real-time integration of knowledge within a drug-development program to support decision making. Here, an R Shiny app was used to explore the pharmacokinetic and pharmacodynamic effects of different dosing regimens of the anti IL-17 human mAb Cosentyx? (secukinumab) in pediatric patients. Secukinumab has been studied and approved to treat psoriasis in adult patients. Models which describe the dose-exposure-response relationships in adults (Lee et al., Clin Pharmacol Ther, 2019 and FDA, Medical Reviews BLA 125504, 2015) were used in the mrgsolve simulation package to explore these relationships in pediatric patients. The prior adult knowledge, used in conjunction with the computational infrastructure leveraged through R, the Shiny app, mrgsolve, and Rcpp, allows researchers to explore various dosing regimens in a difficult-to-study patient population. The tools and approaches described here have been routinely used to support regulatory interactions (ex. PIP) involving pediatric dosing.</p><br>[Slides](https://github.com/rinpharma/rinpharma2019program/tree/master/talks_folder/2019-Fisher-Pediatric_Dosing_Decisions.pdf)</details><br>
<strong>Keaven Anderson</strong> (<i>Merck</i>), Teaching an old dog new tricks: modernizing gsDesign<details><summary>Abstract</summary></p>The gsDesign package for group sequential design is widely used with >30k downloads. The package was originally written in 2007 with substantial documentation and Runit testing created before 2010. A Shiny interface was created to make the package more approachable in about 2015. Recent efforts have focused on updating package to use Roxygen2, pkgdown, covr/covrpage and testthat as well as changing vignettes from Sweave to R Markdown. The learning curve for this modernization will be discussed as well as usage in a regulated environment.</p><br>[Slides](https://github.com/rinpharma/rinpharma2019program/tree/master/talks_folder/2019-Anderson-Modernizing_gsDesign.pptx)</details><br>
<strong>Kelly O'Briant</strong> (<i>RStudio</i>), Shiny in Production: Building bridges from data science to IT<details><summary>Abstract</summary></p>We know that adopting documentation, testing, and version control mechanisms are important for creating a culture of reproducibility in data science. But once you've embraced some basic development best practices, what comes next? What does it take to feel confident that our data products will make it to production? This talk will cover case studies in how I work with R users at various organizations to bridge the gaps that form between development and production. I'll cover reasons why CI/CD tools can enhance reproducibility for R and data science, showcase practical examples like automated testing and push-based application deployment, and point to simple resources for getting started with these tools in a number of different environments.</p><br>[Slides](https://speakerdeck.com/kellobri/shiny-in-production-building-bridges-from-data-science-to-it)</details><br>
